src/gamedata/r_defs.h

```diff
@@ -1644,6 +1644,8 @@ struct subsector_t
 	FPortalCoverage	portalcoverage[2];
 
 	LightmapSurface *lightmap[2];
+
+	TArray<struct particle_t *> particles;
```

Comment:
Looking through the code, I've only ever seen this referenced when pushing a particle upon its creation. How is this being maintained?

Because this looks like it could be a memory leak.